packages/editor-ui/src/components/ParameterInputOverrides/ParameterOverrideSelectableList.vue

```diff
@@ -0,0 +1,72 @@
+<script setup lang="ts">
+import type { IUpdateInformation } from '@/Interface';
+
+import { type INodeProperties } from 'n8n-workflow';
+import { type ParameterOverride } from './parameterInputOverrides';
+
+type Props = {
+	parameter: INodeProperties;
+	path: string;
+	isReadOnly?: boolean;
+};
+
+const parameterOverride = defineModel<ParameterOverride>({ required: true });
+
+const props = withDefaults(defineProps<Props>(), {
+	isReadOnly: false,
+});
+const emit = defineEmits<{
+	update: [value: IUpdateInformation];
+}>();
+
+function valueChanged(parameterData: IUpdateInformation) {
+	emit('update', parameterData);
+}
+</script>
+
+<template>
+	<N8nSelectableList
+		v-model="parameterOverride.extraPropValues"
+		:class="$style.overrideSelectableList"
+		:inputs="
+			Object.entries(parameterOverride.extraProps).map(([name, prop]) => ({
+				name,
+				...prop,
+			}))
+		"
+		:disabled="isReadOnly"
+	>
+		<template #displayItem="{ name, tooltip, initialValue, type, typeOptions }">
+			<ParameterInputFull
+				:parameter="{
+					name,
+					displayName: name[0].toUpperCase() + name.slice(1),
```

Comment:
Can you please clarify why this is needed here?

Reply:
I am just not clear on why proper case. Sorry if I forgot, it's been a minute.

Yes, maybe a comment to explain why this is needed in proper case would help the next person.